vnext/Desktop/DevSupportManager.cpp

```diff
@@ -55,60 +55,50 @@ string DevSupportManager::GetJavaScriptFromServer(
     const string &platform) {
   auto bundleUrl =
       DevServerHelper::get_BundleUrl(debugHost, jsBundleName, platform /*platform*/, "true" /*dev*/, "false" /*hot*/);
-
-  Url url(bundleUrl);
-  auto const resolveResult = m_resolver.resolve(url.host, url.port);
-  tcp::socket socket{m_context};
-  boost::system::error_code ec;
-  connect(socket, resolveResult, ec);
-  if (ec) {
+  Uri uri(Common::Unicode::Utf8ToUtf16(bundleUrl));
+  Filters::HttpBaseProtocolFilter filter;
+  filter.CacheControl().ReadBehavior(Filters::HttpCacheReadBehavior::NoCache);
+  HttpClient client(filter);
+  client.DefaultRequestHeaders().Host(HostName(uri.Host().c_str()));
+
+  HttpRequestMessage request(HttpMethod::Get(), uri);
+  HttpResponseMessage response = client.SendRequestAsync(request).get();
+
+  IBuffer buffer0 = response.Content().ReadAsBufferAsync().get();
+  auto reader = DataReader::FromBuffer(buffer0);
+  reader.UnicodeEncoding(UnicodeEncoding::Utf8);
+  auto length = reader.UnconsumedBufferLength();
+  string result;
+  if (length > 0 || response.IsSuccessStatusCode()) {
+    vector<uint8_t> data(length);
+    reader.ReadBytes(data);
+    result = string(reinterpret_cast<char *>(data.data()), data.size());
+  } else {
     m_exceptionCaught = true;
-    return R"({"error:")"s + ec.message() + R"("})"s;
+    ostringstream stream;
+    stream << R"({"error:")"s << static_cast<int>(response.StatusCode()) << " downloading " << bundleUrl << R"("})"s;
```

Comment:
>s [](start = 28, length = 1)

Should we avoid std::string creation for these string constants here? I would expect that the string constants would be just fine. I.e. the same string without the suffix 's'.

Reply:
Done.